tests/runlib.rs

```diff
@@ -131,6 +131,35 @@ fn in_toto_run_record_new_file() {
     dir.close().unwrap();
 }
 
+#[test]
+fn in_toto_run_new_line_in_stdout() {
```

Comment:
Is this directly tied to the use of utf8? Just checking if I'm missing something.

Reply:
Sorry it is, but I named it very poorly :( 

This is trying to simulate how I ran into this issue and this test will fail without the other changes in this PR. I had a `clone-project` step which cloned a project, and I noticed that the signatures would be different when verifying which was the motivation for opening this PR.